ui/src/app/shared/header/header.component.ts

```diff
@@ -279,16 +279,31 @@ export class JobsPaginatorIntl extends MatPaginatorIntl {
   }
 
   getRangeLabel = (page: number, pageSize: number, length: number) => {
-    if (this.backendJobs.value.exhaustive) {
+
+    let knownLength = null;
+
+    if (this.backendJobs.value.totalKnownResults != null) {
```

Comment:
This line scares me a bit. See this page: http://zero.milosz.ca/

I think perhaps you should more clearly define the expected type union of `JobListView.totalKnownResults`, and then do a `!==` accordingly.